<?xml version="1.0"?>
<launch>
	<!-- Specify either frame_rate or frame_size, if both are specified, the frame_rate will be used. Default value is 100Hz. -->
	<arg name="frame_rate" default=""/>

	<!-- Specify either frame_rate or frame_size, if both are specified, frame_size will be ignored. If none is specified, the default frame_rate of 100Hz will be used. -->
	<arg name="frame_size" default=""/>

	<!-- Specify the input channel count. If no value is given, the maximum input channels of the device will be used. -->
	<arg name="max_channels" default=""/>

	<!-- Specify the file to be used as input. -->
	<arg name="file_name" default=""/>

	<node name="audiofile_transport_publisher" pkg="portaudio_transport" type="audiofile_transport_publisher" output="screen">
		<param name="frame_rate" value="$(arg frame_rate)"/>
		<param name="frame_size" value="$(arg frame_size)"/>
		<param name="max_channels" value="$(arg max_channels)"/>
		<param name="file_name" value="$(arg file_name)"/>
	</node>
</launch>
